### DISCUSSION TITLE
Investigating monitoring bore head response with nearby spring flow

### DISCUSSION THREAD

**Discussion by laat0003** [pastas/pastas#270] [Q&A] [answered]
Hi,
I am considering using Pastas to investigate potential relationships between monitoring bore time series and nearby (>1km) spring discharge. The current operating hypothesis is that there is a 1:1 relationship between reduction in head at a monitoring bore and reduction to spring flow but this has never been demonstrated adequately. These are effectively desert springs fed by artesian aquifers. Is it worthwhile attempting this with Pastas or am I better off considering a black box approach, with say Recurrent Neural Network? If yes to Pastas, would I use the monitoring bore time series as my stress signal and spring discharge as my observation?
Apologies upfront if this is the wrong place to post this type of question.

**Comment by raoulcollenteur** [comment]
Hello! This is the right place, this is exactly what we added Discussions for! 

I think this would be a good use case for Pastas, at least it is worth giving it a try. Whether or not you use the spring discharge or the monitoring bore time series (heads?) as observation or stress depends on which time series you want to model. For example, if you want to model/predict the spring discharge, you can use the monitoring bore time series as a stress (ps.StressModel should work).

Hope this helps! 
Cheers, Raoul

**Reply by laat0003** [reply]
Many thanks for the rapid response. I am working on it today. If successful, this will be the second project that I have used Pastas with. The first was untangling the effects of mine dewatering from regional climate driven groundwater declines. 

Thanks again,
Tariq

**Comment by mbakker7** [comment]
Interesting question.

I would definitely attempt it with Pastas. Since you state that the springs
are fed by the aquifer, I would use the spring discharge as the observation
and the monitoring bore as the stress series. Start out with the
exponential response function to make things easy. If that works ok, you
can switch to Gamma and maybe even the FourParam function. You may need to
twiggle the initial values of the parameters a bit to find a solution,
because the default initial parameters are set up for
different observations and stresses, so they may not be as appropriate for
your case.

**Reply by laat0003** [reply]
Thanks for the tips. They are very much appreciated. Fingers crossed it all works out. If it doesn't work, can I conclude that the change in head at the monitoring bore is not indicative of change in flux at the spring? Or do I have to perform further analysis to make that conclusive? 
My understanding of the methods used in Pastas is that they don't allow for the possibility of interaction between the two time series. For example, head increase at the monitoring bore increases flux at the spring but the increased flux at the spring also reduces the head increase at the bore. From what I've recently read, vector auto-regression is well suited to this type of analysis.

Cheers,
Tariq

**Comment by mbakker7** [comment, marked as answer]
Dear Tariq,

You proposed to make a Pastas model with the measured head in the bore as
stress and the spring flow as observed quantity. If you don't get a good
match, the only conclusion is that the spring flow can not be simulated
with the measured head in the bore as stress and the *selected response
function*  But a model with another response function may give good
results. Or a non-linear model (for example like the non-linear recharge
model) may give good results.

Hope this helps,

Mark Bakker

**Reply by laat0003** [reply]
Cheers for that Mark and apologies for the stupid question. I am aware that different response functions need to be tried and tested. I guess my question was indirectly attempting to ascertain the likelihood of an error on my part being the reason for failing to achieve a suitable fit with Pastas. I usually operate under the assumption that if things don't work then it must be because I have done something wrong. 

Thanks again for your time and advice. Very much appreciate it.
Tariq
